common/pb/edge_cmd_pb/edge_cmd.proto

```diff
@@ -68,6 +68,9 @@ message Authenticator {
     bool isExtendRequested = 6;
     bool isKeyRollRequested = 7;
     google.protobuf.Timestamp extendRequestedAt = 8;
+    string publicKeyPrint = 9;
+    bool lastExtendPublicKeyChanged = 10;
```

Comment:
lastExtendPublicKeyChanged is different than extendRequestedAt, I presume? It'd be nice to keep those names consistent imo and keep the "At"... lastExtendPublicKeyChangedAt

Reply:
One is a bool, the other is a timestamp. They aren't the same?